### PR TITLE
Add recipe for pymt_gipl

### PR DESCRIPTION
**Pull request by mdpiper**
This PR adds a recipe for *pymt_gipl*, a Python wrapper around the [Geophysical Institute Permafrost Laboratory (GIPL) model](https://github.com/conda-forge/gipl-feedstock). 

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pymt_gipl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+    - model_metadata
+    - bmi-fortran =1.2
```
Comment (marcelotrevisani) [inline recipes/pymt_gipl/meta.yaml]: That should be
```suggestion
    - bmi-fortran 1.2
```

File: recipes/pymt_gipl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - gipl
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - bmi-fortran =1.2
```
Comment (marcelotrevisani) [inline recipes/pymt_gipl/meta.yaml]:
```suggestion
    - bmi-fortran 1.2
```

File: recipes/pymt_gipl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pymt-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: bf96a2c99d6a0b0fb577570bb192cbe9b98a686a9ae79590b031a483e41d1953
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (marcelotrevisani) [inline recipes/pymt_gipl/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Comment by mdpiper** [comment]
Thanks for the review, @marcelotrevisani!

**Comment by mdpiper** [comment]
Hi @conda-forge/staged-recipes -- I think this recipe is ready to go, when you have a moment. Thanks!

**Pull request merged**